swap/src/config.rs

```diff
@@ -1 +1,149 @@
+use crate::fs::ensure_directory_exists;
+use anyhow::{Context, Result};
+use config::{Config, ConfigError};
+use dialoguer::{theme::ColorfulTheme, Input};
+use serde::{Deserialize, Serialize};
+use std::{
+    ffi::OsStr,
+    fs,
+    path::{Path, PathBuf},
+};
+use tracing::info;
+use url::Url;
+
 pub mod seed;
+
+const DEFAULT_BITCOIND_TESTNET_URL: &str = "http://127.0.0.1:18332";
+const DEFAULT_MONERO_WALLET_RPC_TESTNET_URL: &str = "http://127.0.0.1:38083/json_rpc";
+
+#[derive(Clone, Debug, serde::Serialize, serde::Deserialize, PartialEq)]
+#[serde(deny_unknown_fields)]
+pub struct File {
```

Comment:
If this module is about defining the `File` struct then it should be called `file.rs`.

If you compare to the other repos, we have
- `config/file.rs`: config file serialization/deserialization
- `config/settings.rs`: the settings to pass around
- `config/mod.rs`: types used in both `file.rs` and `settings.rs`.

Reply:
I combined all relevant config specific code in the module `config.rs` (which serves as module because we use module name over folders and `mod.rs`). The `config.rs` module does not only contain the `File` struct, but also functions to read and initialize the config. I can separate `File` into a separate `file` module under `config`, but I did not see the necessity yet.